testing/sdk_fault_domain.py

```diff
@@ -29,17 +29,23 @@
 ]
 
 # TODO: use cloud provider library APIs to get list of zones.
-AWS_ZONE_SUFFIXES = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h']
+# a through h
+AWS_ZONE_SUFFIXES = [chr(i) for i in range(ord('a'), ord('h')+1)]
 
 
+# expect e.g. "aws/us-west-2" or "ca-central-1"
 def is_valid_aws_region(region: str):
+    if region.startswith('aws/'):
```

Comment:
@nickbp Are you sure it doesn't start with `/aws/`?

Reply:
The failure I saw lacked a preceding `/`

Reply:
Nevermind:
```
domain: {
	fault_domain: {
		region: {
			name: "aws/us-west-2"
		},
		zone: {
			name: "aws/us-west-2a"
		}
	}
}
```